src/systemd/nm-sd-utils.c

```diff
@@ -0,0 +1,45 @@
+/* This library is free software; you can redistribute it and/or
```

Comment:
I like the idea of reusing code, but I'm worried that this will make imports of systemd code more complex when the internal systemd API changes.

 As long as we reuse only few functions, I think this is useful.

Reply:
Agree. I mentioned, that this must be used carefully. Also, all the functions that I currently exposed are arguably trivial (why would systemd change them?) and they have unit-tests in NM. The rule should be, that such code must have unit-tests.